src/mpid/ch4/shm/posix/eager/iqueue/iqueue_init.c

```diff
@@ -101,7 +101,7 @@ int MPIDI_POSIX_iqueue_init(int rank, int size)
     int nprocs = MPIR_Process.local_size;
 
     int pool_size = MPIDU_genq_shmem_pool_size(cell_size, num_cells, nprocs, num_free_queue);
-    int terminal_size = num_proc * sizeof(MPIDU_genq_shmem_queue_u);
```

Comment:
looks like a leftover from some other commit, not related to [9577a84ab10529465b1d02ffc12590ba421acd4f]